src/main/openapi/openapi_v1.yaml

```diff
@@ -227,10 +227,660 @@ paths:
                 type: array
                 items:
                   $ref: '#/components/schemas/Collection'
+  
+# IIIF Presentation endpoints
+  /IIIF/{identifier}/collection/{name}:
```

Comment:
This looks... wrong? Is the [collections](https://iiif.io/api/presentation/2.0/#collections) (plural)? And if so, I don't think there should be an `identifier`?

Reply:
Correct! It is indeed plural and it does not contain an `identifier`